### ISSUE TITLE
Issue with using items on >= 1.17 versions

### ISSUE THREAD

**Issue by Cosmic-SC** [ViaVersion/ViaForge#79]
### General Information

viaforge latest with optifine.
pretty sure this only happens if your actual client version is below 1.17

### Bug Description

Some servers kick for "invalid item use" or just a straight-up ban if you are using any version that is above or equal to 1.17.
After checking a bit, I think the issue is related to "placeBlockDirection" in the placement packet; the kick or ban only happens if the value of "placeBlockDirection" is 255.

### Steps to Reproduce

join gamster (with version changed to 1.17 or above) -> join practice or bedwars or skywars -> use a consumable item.
practice and bedwars kick you while in skywars, you either get banned or kicked after some time.

### Expected Behavior

you should not get kicked or banned for using items :(((

**Comment by FlorianMichael** [comment]
I don't think this will be ever implemented in ViaForge, Mojang has changed quite a lot stuff in those versions and I don't have any interests in implementing more clientside related fixes into ViaForge.

**Comment by Cosmic-SC** [comment]
alright 👍

**Comment by FlorianMichael** [comment]
I can't recommend anyone to use old Minecraft versions or mod loaders, I can recommend ViaFabricPlus which is a Fabric 1.20.4 mod adding support for every Minecraft multiplayer version and it even contains tons of improvements to the game play like 1.8 combat and animations.

**Comment by Cosmic-SC** [comment]
> I can't recommend anyone to use old Minecraft versions or mod loaders, I can recommend ViaFabricPlus which is a Fabric 1.20.4 mod adding support for every Minecraft multiplayer version and it even contains tons of improvements to the game play like 1.8 combat and animations.

unfortunately I work mainly on 1.8 stuff and I don't have enough time to learn protocol for newer versions :(
